specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-06-01-preview/PrivateEndpointConnections.json

```diff
@@ -0,0 +1,270 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved a specified private endpoint connection.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets private endpoint connection.": {
+            "$ref": "./examples/PrivateEndpointConnectionGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "description": "Approve or reject a private endpoint connection with a given name.",
+        "operationId": "PrivateEndpointConnection_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../types/common-types.json#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/parameters/PrivateEndpointConnectionName"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnection"
+            },
+            "description": "The required parameters for updating private endpoint connection."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully approved or rejected private endpoint connection.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "201": {
+            "description": "Successfully approved or rejected private endpoint connection.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "202": {
+            "description": "Accepted",
```

Comment:
[ArmBlockingComment] 202 is not allowed for a long running Put

Reply:
THis is what I mentioned in the comment provided earlier and also sent you an email. We have a work-item “[Product Backlog Item 2760721](https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/2760721): [PG Flex] New PUT APIs for PostgreSQL flexible server should not have 202 Accepted as response” created to handle this but our feature commitments cannot wait until this gets fixed. Entire framework is currently running the same way. For new APIs to work as per the latest guidelines we will need to make some changes at our end to make sure that earlier APIs do not get affected and new ones will work properly as per the guidelines. I had also raised this couple of weeks back on Teams channel [Amit Brahma: [ARM Query] PUT operation supports 202 Accepted](https://teams.microsoft.com/l/message/19:0351f5f9404446e4b4fd4eaf2c27448d@thread.skype/1697462898804?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1697462898804&teamName=Azure%20SDK&channelName=API%20Spec%20Review&createdTime=1697462898804) and on stackoverflow https://stackoverflow.microsoft.com/questions/374232 but there was no response.

Reply:
As discussed during the office hrs, our existing flexibleServers namespace is using this old pattern to return 202. We have a workitem to address this, but this API is under same namespace and hence added suppression for this code readme file

Reply:
Allowing this since there are other resource types following the same pattern and already shipped.